Source/WebKit/UIProcess/API/Cocoa/_WKWebExtensionMatchPattern.mm

```diff
@@ -175,16 +175,22 @@ - (NSString *)debugDescription
 
 - (NSString *)scheme
 {
+    if (_webExtensionMatchPattern->scheme().isNull())
+        return nil;
     return _webExtensionMatchPattern->scheme();
 }
 
 - (NSString *)host
 {
+    if (_webExtensionMatchPattern->host().isNull())
+        return nil;
     return _webExtensionMatchPattern->host();
 }
 
 - (NSString *)path
 {
+    if (_webExtensionMatchPattern->path().isNull())
+        return nil;
     return _webExtensionMatchPattern->path();
```

Comment:
It seems like these three methods could use a local to get their relevant variables, but definitely not necessary.